.circleci/config.yml

```
@@ -49,10 +46,7 @@ jobs:
       - run:
           name: Install test data from GIN
           command: |
-              export PY3=$(pyenv versions | grep '3\.' |
-                           sed -e 's/.* 3\./3./' -e 's/ .*//')
-              pyenv local $PY3
-              export PATH=/opt/circleci/git-annex.linux:$PATH
+              export PATH=/opt/circleci/.pyenv/versions/3.12.2/bin/:/opt/circleci/git-annex.linux:$PATH
```

Rather than fixing to a version, I think it'll be easier to just use the latest available
```suggestion
              pyenv local 3
```

Then you wouldn't need to fiddle with the cache